src/MatricesForHomalg.jl

```diff
@@ -1330,46 +1330,52 @@ function LeftDivide(A, B)::Union{TypeOfMatrixForHomalg, String}
 end
 
 """
-    DecideZeroRows(mat1, mat2)
+    DecideZeroRows(B, A)
+
+    Returns: a homalg matrix
+
+Let A and B be matrices having the same number of columns and defined over the same ring R and S be the row span of B, i.e. the R-submodule of the free left module R(1×NrColumns(B)) spanned by the rows of B.
+The result is a matrix C having the same shape as A, for which the i-th row Ci is equivalent to the i-th row Ai of A modulo S, i.e. Ci−Ai is an element of the row span S of B.
```

Comment:
The documentation is wrong, but we can fix this in a future PR.

Reply:
Done